### PR TITLE
basic tabs

### PR DESCRIPTION
**Pull request by YevhenLukomskyi**

### CODE REVIEW

File: assets/tasks/tabs/index.jsx [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+const activeTabId = 2;
+
+
+class App extends React.Component {
+  render() {
+    return (
+      <Tabs tabs={tabs} activeTabId={activeTabId} />
+    );
+  }
+}
```
Comment (mykhailo-riabokon) [inline assets/tasks/tabs/index.jsx]: Can you use Stateless component instead?

File: assets/tasks/tabs/tab_body.jsx [hunk 2]
```diff
@@ -0,0 +1,14 @@
+function TabBody(props) {
+  return (
+    <div className={props.isActive === true ? "ui bottom attached tab segment active" : "ui bottom attached tab segment"}
```
Comment (mykhailo-riabokon) [inline assets/tasks/tabs/tab_body.jsx]: Maybe better to create some variable for this?

File: assets/tasks/tabs/tabs.jsx [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+    return (
+      <div className="ui container">
+        <div className="ui top attached tabular menu">
+          {
+            this.props.tabs.map((tab) => {
+              return <TabHeader key={tab.id}
+                id={tab.id}
+                title={tab.title}
+                isActive={tab.id === this.state.activeTabId}
+                setActive={this.setActiveTab.bind(this, tab.id) }/>
```
Comment (mykhailo-riabokon) [inline assets/tasks/tabs/tabs.jsx]: What if you bind it in constructor?

**Comment by YevhenLukomskyi** [comment]
@mikhail-riabokon Updated

**Comment by mykhailo-riabokon** [comment]
Done!

**Pull request closed without merging**